core/templates/dev/head/pages/collection_editor/collection_editor_navbar_directive.html

```diff
@@ -55,14 +55,14 @@
     </li>
   </ul>
 
-  <ul class="nav navbar-nav oppia-navbar-nav navbar-right">
+  <ul class="nav navbar-nav oppia-navbar-nav navbar-right" ng-if="!isPageLoading()">
```

Comment:
Do we have some other loading indicator controlled by ``isPageLoading()``? Otherwise, when this is false, what shows up?